e2etest/templates/performance_model.tpl

```diff
@@ -8,9 +8,9 @@
 
 {% for models in models_list %}
 ### {{ models.data_mode }} (TPS: {{ models.data_rate }})
-| Test Case |  Data Type | Instance Type | Avg CPU Usage (Percent) | Avg Memory Usage (Megabytes) |
-|:---------:|:----------:|:------------:|:-----------------------:|:----------------------------:|
+| Test Case | Receivers | Processors | Exporters | Data Type | Instance Type | Avg CPU Usage (Percent) | Avg Memory Usage (Megabytes) |
```

Comment:
can we sort the list by receivers name instead of testcase name?

Reply:
Sure, do we want the pipeline to come first then or keep the column ordering the same?

Reply:
And in that case, do we also want to sort by processors and exporters as well, before testcase name?

Reply:
we can just sort on the receiver at this moment

Reply:
and yes components go first